apps/web/src/services/asset-inventory/collector/shared/CollectorPluginContents.vue

```diff
@@ -55,63 +63,58 @@ import {
     screens,
 } from '@spaceone/design-system';
 
-import { store } from '@/store';
-
-import type { PluginReferenceItem, PluginReferenceMap } from '@/store/modules/reference/plugin/type';
+import { assetUrlConverter } from '@/lib/helper/asset-helper';
 
 import { repositoryColorMap, repositoryIconMap, repositoryBackgroundColorMap } from '@/services/asset-inventory/collector/config';
 import type {
-    CollectorPluginModel,
-    RepositoryInfo,
     RepositoryPluginModel,
 } from '@/services/asset-inventory/collector/model';
 
 type Size = 'sm' | 'lg';
 
 // TODO: Add plugin data type
```

Comment:
question: Since there's a `plugin` data type, is it safe to delete the TODO?

Reply:
Oh, I missed it. thank you~